x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/geo_field/description.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiDescribedFormGroup, EuiFormRow } from '@elastic/eui';
+
+export const Description: FC = memo(({ children }) => {
+  const title = i18n.translate('xpack.ml.newJob.wizard.pickFieldsStep.geoField.title', {
+    defaultMessage: 'Geo field',
+  });
+  return (
+    <EuiDescribedFormGroup
+      title={<h3>{title}</h3>}
+      description={
+        <FormattedMessage
+          id="xpack.ml.newJob.wizard.pickFieldsStep.geoField.description"
+          defaultMessage="Select a geo field to detect anomalies in the geographic location of the input data."
```

Comment:
>          defaultMessage="Select a geo field to detect anomalies in the geographic location of the input data."

Per https://www.elastic.co/guide/en/machine-learning/current/ml-geo-functions.html, "The field_name that you supply must be a single string that contains two comma-separated numbers of the form latitude,longitude, a geo_point field, a geo_shape field that contains point values, or a geo_centroid aggregation. The latitude and longitude must be in the range -180 to 180 and represent a point on the surface of the Earth."  I presume this wizard filters the possible field values to only those that meet these requirements. If not, however, it might be helpful to include this type of information in a tip.

Reply:
The wizard does only show supported fields already - and also the wizard option only shows up if there are fields that are supported by this type of job so the user wouldn't be running into any situations in which there would be no fields in the dropdown.
🤔 We could still add a tooltip with info about the fields just for information sake, though I don't think it's necessary for this PR. Happy to add in follow up